src/python_bindings/bindings.cpp

```diff
@@ -156,7 +156,57 @@ PYBIND11_MODULE(desbordante, module) {
             .def("get_num_error_rows", &PyFDVerifier::GetNumErrorRows)
             .def("get_highlights", &PyFDVerifier::GetHighlights);
 
-    DEFINE_ALGORITHM(DataStats, Algorithm).def("get_result_string", &PyDataStats::GetResultString);
+    DEFINE_ALGORITHM(DataStats, Algorithm)
+            .def("get_result_string", &PyDataStats::GetResultString)
+            .def("get_number_of_values", &PyDataStats::GetNumberOfValues,
+                 "Get number of values in column.")
+            .def("get_number_of_columns", &PyDataStats::GetNumberOfColumns,
+                 "Get number of columns in table.")
+            .def("get_columns_with_null", &PyDataStats::GetColumnsWithNull,
+                 "Get number columns with null.")
+            .def("get_columns_with_unique_values", &PyDataStats::GetColumnsWithUniqueValues,
+                 "Get number of columns with unique values.")
```

Comment:
```suggestion
            .def("get_columns_with_all_unique_values", &PyDataStats::GetColumnsWithUniqueValues,
                 "Get indices of columns where all values are distinct.")
```
There has to be a name for this type of column, but I don't know it.

Reply:
It does consider all possible types of columns

Reply:
I think `get_unique_columns` should work, referring to the `UNIQUE` constraint in databases. Probably not ideal, but whatever.

Reply:
Unique in this context refers more to the columns that are unique among all other columns in a table